components/layout/fragment.rs

```diff
@@ -2550,6 +2541,18 @@ impl Fragment {
         if self.style().get_effects().transform.0.is_some() {
             return true
         }
+
+        // TODO(mrobinson): Determine if this is necessary, since blocks with
+        // transformations already create stacking contexts.
+        if self.style().get_effects().perspective != LengthOrNone::None {
```

Comment:
Do you mind if I remove this in a followup change, since I want to minimize the risk of changing behavior in this PR?